spec/site_prism/page_objects/page/result_filters/location.rb

```diff
@@ -0,0 +1,14 @@
+module PageObjects
+  module Page
+    module ResultFilters
+      class Location < SitePrism::Page
+        set_url "/results/filter/location{?query*}"
```

Comment:
```suggestion
        set_url "/results/filter/location{?test%5B%5D*}"
```

Took a bit of digging to sort this out, but basically in the RFC for these URL templates, `query` here is an [`expression`](https://tools.ietf.org/html/rfc6570#section-2.2) which is constituted of a [`variable-list`](https://tools.ietf.org/html/rfc6570#section-2.3), which may include `pct-encoded` elements that, in the "[notation convention](https://tools.ietf.org/html/rfc6570#section-1.5)", escapes reserved characters (such as "[" and "]") using hex digits. Which means `test[]` has to be escaped as `test%5B%5D`. In my testing, this seems to work.

Reply:
`query` is used in the tests to load pages with a specific query - although arrays can be a bit more finicky. But we `query` in all the other tests and changing it to this would break all of those

`page.load(query: { foo: 'bar' })` is an example of how we use this